lib/timescaledb/scenic/extension.rb

```diff
@@ -67,5 +67,5 @@ def create_scenic_continuous_aggregate(name)
 end
 
 
-Scenic::Adapters::Postgres.include(Timescaledb::Scenic::Extension)
+Scenic::Adapters::Postgres.prepend(Timescaledb::Scenic::Extension)
```

Comment:
For some reason, after updating the version, the method override did not recognize the signature change, and the `prepend` fixed the load order.